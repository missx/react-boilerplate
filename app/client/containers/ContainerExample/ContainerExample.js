import React from 'react';
import { Fragment } from 'redux-little-router';

class ContainerExample extends React.Component {

	constructor(props) {
		super(props);
	}

	render() {
		return (
			<div className="align">
				<h1>{this.props.text}</h1>
			</div>
		);
	}
}

export default ContainerExample;