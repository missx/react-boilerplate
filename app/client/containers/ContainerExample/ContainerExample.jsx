import React from 'react';

export default class ContainerExample extends React.Component {
  render() {
    return (
      <div className="align">
        <h1>Container</h1>
      </div>
    );
  }
}