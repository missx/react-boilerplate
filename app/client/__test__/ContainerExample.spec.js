import React from 'react';
import renderer from 'react-test-renderer';

import ContainerExample from '../containers/ContainerExample/ContainerExample';

let text = 'This is a test!';

it('renders correctly', () => {
    const tree = renderer.create(
        <ContainerExample text={text}></ContainerExample>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
