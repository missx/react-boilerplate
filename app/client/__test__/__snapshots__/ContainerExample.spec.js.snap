// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="align"
>
  <h1>
    This is a test!
  </h1>
</div>
`;
